cli/names.go

```diff
@@ -0,0 +1,19 @@
+package cli
+
+import (
+	"strings"
+
+	"github.com/docker/cli/cli/names"
+)
+
+var (
+	validContainerNamePattern = names.RestrictedNamePattern
```

Comment:
I think the package "names" you added is not very useful. Why not putting the regexp and the pattern definition directly here in this file? It will also remove the need to expose the pattern and the regexp.

Reply:
Yes, it's just from moby project. I'll remove it.